projects/aqlprofile/src/core/include/aqlprofile-sdk/version.h.in

```diff
@@ -0,0 +1,121 @@
+// MIT License
+//
+// Copyright (c) 2025 Advanced Micro Devices, Inc. All rights reserved.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#pragma once
+
+/**
+ * @def AQLPROFILE_VERSION_MAJOR
+ * @brief The major version of the interface as a macro so it can be used
+ * by the preprocessor.
+ * @addtogroup VERSIONING_GROUP
+ *
+ * @def AQLPROFILE_VERSION_MINOR
+ * @brief The minor version of the interface as a macro so it can be used
+ * by the preprocessor.
+ * @addtogroup VERSIONING_GROUP
+ *
+ * @def AQLPROFILE_VERSION_PATCH
+ * @brief The patch version of the interface as a macro so it can be used
+ * by the preprocessor.
+ * @addtogroup VERSIONING_GROUP
+ *
+ * @def AQLPROFILE_VERSION_NPI
```

Comment:
Why is there an NPI version number?

Reply:
This should help with differentiating OSS and NPI builds. It's a non-zero number for NPI builds. Do I remove it?

Reply:
Removed